bin/torii/build.rs

```diff
@@ -0,0 +1,30 @@
+use std::env;
+use std::error::Error;
+
+use vergen::{BuildBuilder, Emitter};
+use vergen_gitcl::GitclBuilder;
+
+fn main() -> Result<(), Box<dyn Error>> {
+    let build = BuildBuilder::default().build_timestamp(true).build()?;
+    let gitcl =
+        GitclBuilder::default().describe(true, true, None).branch(true).sha(true).build()?;
+
+    // Emit the instructions
+    Emitter::default().add_instructions(&build)?.add_instructions(&gitcl)?.emit_and_set()?;
+
+    let version = env!("CARGO_PKG_VERSION");
+    let git_branch = env::var("VERGEN_GIT_BRANCH").unwrap_or("unknown".to_string());
+    let git_sha = env::var("VERGEN_GIT_SHA").unwrap_or("unknown".to_string());
+    let git_describe = env::var("VERGEN_GIT_DESCRIBE").unwrap_or("unknown".to_string());
+
+    let dev = git_describe.contains(&git_sha) || git_describe.contains("dirty");
+
+    let version = if dev {
```

Comment:
What's the use case of `dev`, mostly to warn user about the fact that it's not the one built by the production CI?

Otherwise, having the branch and the commit is helpful.